CMake error `file INSTALL cannot find "/root/catkin_ws/src/ouster-ros/../LICENSE".`

Hi!

We use `ouster-ros` in our build process, and hadn't pinned a specific commit so our builds stopped working this morning.
After some time, we realized it was because you split this library out of the old repository (good change!), so I've been updating our builds but when I run `catkin_make install`, I get this error message:
```
file INSTALL cannot find "/root/catkin_ws/src/ouster-ros/../LICENSE".
Call Stack (most recent call first):
cmake_install.cmake:136 (include)

Makefile:73: recipe for target 'install' failed
```

I assume that there is some error in the [`CMakeLists.txt`](https://github.com/ouster-lidar/ouster-ros/blob/master/CMakeLists.txt), maybe the `../` should be removed from `../LICENSE`?

EDIT: `LICENSE-bin` doesn't exist in this repository either, so should that line be removed?

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -2,6 +2,10 @@
 Changelog
 =========
 
+[Unreleased]
+============
+* Update LICENSE installation.
+
 [20221004]
 ==========
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -105,8 +105,7 @@ install(
 
 install(
   FILES
-    ../LICENSE
-    ../LICENSE-bin
+    LICENSE
     nodelets_os.xml
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )

--- a/Dockerfile
+++ b/Dockerfile
@@ -49,7 +49,8 @@ RUN set -xe \
 FROM build-env
 
 ENV CXXFLAGS="-Werror -Wno-deprecated-declarations"
-RUN /opt/ros/$ROS_DISTRO/env.sh catkin_make -DCMAKE_BUILD_TYPE=Release
+RUN /opt/ros/$ROS_DISTRO/env.sh catkin_make -DCMAKE_BUILD_TYPE=Release \
+&& /opt/ros/$ROS_DISTRO/env.sh catkin_make install
 
 # Entrypoint for running Ouster ros:
 #

--- a/launch/record.launch
+++ b/launch/record.launch
@@ -16,7 +16,8 @@
     512x20,
     1024x10,
     1024x20,
-    2048x10
+    2048x10,
+    4096x5
     }"/>
   <arg name="timestamp_mode" default="" doc="method used to timestamp measurements; possible values: {
     TIME_FROM_INTERNAL_OSC,

--- a/launch/sensor.launch
+++ b/launch/sensor.launch
@@ -16,7 +16,8 @@
     512x20,
     1024x10,
     1024x20,
-    2048x10
+    2048x10,
+    4096x5
     }"/>
   <arg name="timestamp_mode" default="" doc="method used to timestamp measurements; possible values: {
     TIME_FROM_INTERNAL_OSC,

